bin/pgo-backrest-repo/pgo-backrest-repo.sh

```diff
@@ -55,6 +59,17 @@ then
 	then
 		printf "repo1-s3-key-secret=%s\n" "${PGBACKREST_REPO1_S3_KEY_SECRET}" >> /etc/pgbackrest/pgbackrest.conf
 	fi
+
+	if [[ "${PGBACKREST_REPO1_S3_URI_STYLE}" != "" ]]
+	then
+		printf "repo1-s3-uri-style=%s\n" "${PGBACKREST_REPO1_S3_URI_STYLE}" >> /etc/pgbackrest/pgbackrest.conf
+	fi
+
+        if [[ "${PGBACKREST_REPO1_S3_VERIFY_TLS}" != "" ]]
+	then
+		printf "repo1-s3-verify-tls=%s\n" "${PGBACKREST_REPO1_S3_VERIFY_TLS}" >> /etc/pgbackrest/pgbackrest.conf
+	fi
```

Comment:
Because we are managing these now, we may want to add these `backup-opts` to the denyList in backrestoptions.go

Reply:
Agreed, that makes sense. Added.